jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -264,7 +270,25 @@ function(params) {
                        readOnly: true,
                      },
                    ] else []
-                 ) else []),
+                 ) else []) + (
+                   if std.objectHas(api.config, 'additionalWriteEndpoints') then [
+                     {
+                       name: 'endpoint-config',
+                       mountPath: '/var/run/config/endpoints.yaml',
+                       subPath: 'endpoints.yaml',
+                       readOnly: true,
+                     },
+                   ] + (
+                    if std.objectHas(api.config.additionalWriteEndpoints, 'mountSecrets') then [
+                      {
+                        name: name,
+                        mountPath: '/var/run/secrets/' + name,
```

Comment:
@marcolan018 It seems here requires the users must configure the certification under this folder `/var/run/secrets/`. Can we get the folder from the endpointconfig secrect?